src/user_templates_api/templates/jupyter_lab/templates/blank/metadata.json

```diff
@@ -1,6 +1,6 @@
 {
-    "title": "Blank notebook",
-    "description": "Notebook only inserting UUIDs of added datasets",
+    "title": "Blank Notebook in Python",
```

Comment:
If we add a distinguishing field like `job_types` we wouldn't need to change the title.